app/Http/Controllers/OrderedItemController.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\CreateOrderRequest;
+use App\Models\OrderedItem;
+use App\Models\Party;
+use App\Services\ForbiddenException;
+use App\Services\NotFoundException;
+use App\Services\OrderedItemService;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\HttpException;
+
+class OrderedItemController extends Controller
+{
+    public function __construct()
+    {
+        $this->middleware('auth:api');
+    }
+
+    /**
+     * 食べる人が注文した全ての料理を返す
+     */
+    public function index(OrderedItemService $orderedItemService)
+    {
+        /* @var Party $party */
+        $party = Auth::user();
+
+        return $orderedItemService->queryByPartyId($party->id);
```

Comment:
[May]
Service層の関数はビジネスロジックを表して欲しいので、 `query` みたいな裏側（DB）を意識するものより、 `get` のような抽象度が高いものが良いと思います。

```suggestion
        return $orderedItemService->getOrderedByPartyId($party->id);
```

Reply:
修正しました！